lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -322,3 +322,11 @@ section.foldit {
 
    }
 }
+
+#courseware-results-wrapper {
+  padding: ($baseline*2);
+
+  #courseware-results-list {
```

Comment:
NIT: if possible, it would be great to use class based styles instead of an ID selector here.  Also, is this for a case of it being empty? Could this be moved into another rule potentially?